OracleDataIntegrator/dockerfiles/12.2.1.2.6/oraInst.loc

```diff
@@ -0,0 +1,2 @@
+inventory_loc=/u01/oracle/.inventory
```

Comment:
I have followed the bestpracticeReferences given to me :) and I can see same in OracleWebLogic Docker files also. :)